src/screens/TransactionConfirmationScreen.js

```diff
@@ -237,6 +238,10 @@ export default function TransactionConfirmationScreen() {
     return isL2Network(network);
   }, [network]);
 
+  const isTestnet = useMemo(() => {
```

Comment:
this is an example of something where bc it is so lightweight, the memo becomes more overhead than help

can just be:
```
const isTestnet = isTestnetNetwork(network);
```
(and yes we're doing this inconsistently across app :( )

Reply:
gonna change isL2 as well

Reply:
this comment still stands